storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -246,7 +248,11 @@ replace below code with split(",") once 'storm.cmd' is fixed to send url-encoded
               */
             String[] configs = commandOptions.split(",(?![^\\[\\]{}]*(]|}))");
             for (String config : configs) {
-                config = URLDecoder.decode(config);
+                try {
+                    config = URLDecoder.decode(config, StandardCharsets.UTF_8.name());
```

Comment:
Could we add this to util class so that we don't need to specify `StandardCharsets.UTF_8.name()` all the time? Maybe two versions throwing original UnsupportedEncodingException and RuntimeException wrapping UnsupportedEncodingException.

Reply:
Good idea. Fixed.